### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by mybearworld** [TodePond/GulfOfMexico#351] [Q&A] [answered]
[audio.webm](https://github.com/TodePond/WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite/assets/130385691/a9bdbb3d-ec51-4e56-bb6c-a22aeed35cdf)

**Comment by TodePond** [comment, marked as answer]
https://github.com/TodePond/WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite/assets/15892272/d6330889-afae-4372-a0dd-c3727d0536d4

**Reply by mybearworld** [reply]
[audio2-why-do-i-need-to-name-these-files-AAAAAAAAAAAAA.webm](https://github.com/TodePond/WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite/assets/130385691/062cbf6e-b34f-4dbf-8a49-f128c5f10db5)
